MANIFEST.in

```
@@ -12,7 +12,7 @@ recursive-include btms_ui *.qss
 recursive-exclude btms_ui *.ui *.png
 
 include versioneer.py
-include btms_ui/_version.py
+include btms_ui/version.py
```

```suggestion
include btms_ui/_version.py
```

I think this is actually still `_version.py` with the underscore.  The `_version.py` file gets autogenerated and packaged for pip / conda installs.

I will not claim this is obvious by any means though, I'm mostly comparing with our working packages 😆